src/rules/maxLineLengthRule.ts

```diff
@@ -29,12 +28,20 @@ export class Rule extends Lint.Rules.AbstractRule {
             Limiting the length of a line of code improves code readability.
             It also makes comparing code side-by-side easier and improves compatibility with
             various editors, IDEs, and diff viewers.`,
-        optionsDescription: "An integer indicating the max length of lines.",
+        optionsDescription: Lint.Utils.dedent`It can take up to 2 arguments, but only first one is required:
+
+        * integer indicating the max length of lines.
+        * string defining ignore pattern, being parsed by \`new RegExp()\`.
```

Comment:
Do you mean object including the first number parameter? How'd you name it? 
Can it accept number or object (like `number|object`) to avoid breaking change?